src/repo/ConferenceRepository.java

```diff
@@ -30,4 +30,19 @@ public ConferenceRepository() {
             return null;
         }
     }
+    
+    public Conference getById(Integer id) {
```

Comment:
Poti sa faci metoda asta generica si sa o pui in BaseRepository te rog?Si numeste-o doar "get"